sys-apps/ccs-tools/ccs-tools-1.8.3_p20130214.ebuild

```diff
@@ -1,31 +1,30 @@
-# Copyright 1999-2017 Gentoo Foundation
+# Copyright 1999-2022 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=8
 
 inherit flag-o-matic toolchain-funcs
 
 MY_P="${P/_p/-}"
 
 DESCRIPTION="TOMOYO Linux tools"
-HOMEPAGE="http://tomoyo.sourceforge.jp/"
+HOMEPAGE="https://tomoyo.sourceforge.jp/"
 SRC_URI="mirror://sourceforge.jp/tomoyo/49693/${MY_P}.tar.gz"
+S="${WORKDIR}/${PN}"
 
 LICENSE="GPL-2"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
-IUSE=""
 RESTRICT="test"
 
 CDEPEND="
```

Comment:
While you are at it, could you rename to ```COMMON_DEPEND```? Just in case, if we ever should introduce some official ```CDEPEND```. Thanks!